### ISSUE TITLE
"honeypots -h" command produces no output after install

### ISSUE THREAD

**Issue by br0ziliy** [qeeqbox/honeypots#42]
I'm having what seems to be the same problem as in #24  - running command "honeypots" produces no output and nothing happens.

Some system info:

```
(honeypot) [root@server ~]# python -V
Python 3.6.8
(honeypot) [root@server ~]# pip3 show honeypots
Name: honeypots
Version: 0.56
Summary: 23 different honeypots in a single pypi package! (dns, ftp, httpproxy, http, https, imap, mysql, pop3, postgres, redis, smb, smtp, socks5, ssh, telnet, vnc, mssql, elastic, ldap, ntp, memcache, snmp, oracle, sip and irc) 
Home-page: https://github.com/qeeqbox/honeypots
Author: QeeqBox
Author-email: gigaqeeq@gmail.com
License: AGPL-3.0
Location: /root/ve/honeypot/lib/python3.6/site-packages
Requires: impacket, netifaces, paramiko, psutil, psycopg2-binary, pycrypto, requests, requests, scapy, service-identity, twisted
Required-by: 
```
It's a CentOS 7 server.

What can I do to troubleshoot it further?

**Comment by br0ziliy** [comment]
I was able to find the issue. The script dies with following traceback:

```
Traceback (most recent call last):
  File "/root/ve/honeypot/bin/honeypots", line 5, in <module>
    from honeypots.__main__ import main_logic
  File "/root/ve/honeypot/lib64/python3.6/site-packages/honeypots/__init__.py", line 19, in <module>
    from .elastic_server import QElasticServer
  File "/root/ve/honeypot/lib64/python3.6/site-packages/honeypots/elastic_server.py", line 20, in <module>
    from http.server import SimpleHTTPRequestHandler, ThreadingHTTPServer
ImportError: cannot import name 'ThreadingHTTPServer'
```

`ThreadingHTTPServer` is Python 3.7 feature, so honeypots won't work on Python < 3.7; I guess package metadata should be updated to indicate this (it currently states that Python >= 3.5 is supported).

Another issue is that the traceback above only visible in debugger (output is written to `/dev/null`)
